File: 2dam/ProgramacionServiciosProcesos/ColaRodrigoTapiador/src/main/java/dataclases/Persona.java
package dataclases;

public class Persona {
    int edad;
    String nombre;
    int prioridad;
    int orden;


    public Persona(int edad, String nombre) {
        this.edad = edad;
        this.nombre = nombre;
    }
    public Persona(int edad, String nombre, int prioridad) {
        this.edad = edad;
        this.nombre = nombre;
        this.prioridad = prioridad;
    }
    public Persona(int edad, String nombre, int prioridad, int orden) {
        this.edad = edad;
        this.nombre = nombre;
        this.prioridad = prioridad;
        this.orden = orden;
    }

    @Override
    public String toString() {
        return "dataclases.Persona{" +
                "edad=" + edad +
                ", nombre='" + nombre + '\'' +
                ", prioridad=" + prioridad +
                ", orden=" + orden +
                '}';
    }
}

File: 2dam/DesarrolloInterfaces/source/repos/WinFormsExamenRodrigoTapiador/Models/dataclases/Persona.cs
namespace WinFormsExamenRodrigoTapiador.Models.dataclases;

public class Persona
{
    public string Nombre { get; set; }
    public int Id { get; set; }


    public Persona() { }
    public Persona(string nombre)
    {
        Nombre = nombre;
    }
}
